BoardGameTrainer/CustomWidgets/StopConditionFrame.cs

```diff
@@ -27,9 +28,9 @@ public StopConditionFrame(Dictionary<string, IStopConditionFactory> entries) : b
             Box.PackStart(Boxes[1], true, true, 5);
 
             Boxes[VisibleBox].Show();
-            DropDown.Changed += (sender, args) => 
-            { 
-                Boxes[VisibleBox].Hide(); 
+            DropDown.Changed += (sender, args) =>
```

Comment:
trzeba się upewnić  czy to changed działa poprawnie

Reply:
Teraz tak